img/f/fortuneo.fr.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.2" baseProfile="tiny" xml:space="preserve" width="39.62921342896698" height="42.5056179052891" style=""><rect id="backgroundrect" width="100%" height="100%" x="0" y="0" fill="none" stroke="none" class="" style=""/>
+
+
+<g class="currentLayer" style=""><title>Layer 1</title><g id="svg_1" class="selected">
+	<path fill="#88C648" d="M13.290962947988652,20.925684813516376 c-0.16167561433860636,-0.211348932339692 -0.5389187144620209,-0.15851169925476902 -0.5389187144620209,0.10567446616984602 L8.60237013216907,42.43043867908004 c-0.05389187144620212,0.31702339850953815 0.37724310012341483,0.5283723308492302 0.5389187144620209,0.211348932339692 l9.646644988870172,-13.314982737400591 c0.16167561433860636,-0.211348932339692 0.16167561433860636,-0.5283723308492302 0,-0.7925584962738448 L13.290962947988652,20.925684813516376 z" id="svg_2"/>
+	<path fill="#88C648" d="M39.751871828073874,11.996192422164393 H27.626200752678404 c-0.21556748578480833,0 -0.3233512286772126,0.15851169925476902 -0.3233512286772126,0.369860631594461 l0.9161618145854353,4.755350977643069 c0.05389187144620212,0.211348932339692 0.26945935723101067,0.31702339850953815 0.43113497156961683,0.211348932339692 l11.209509260810036,-4.755350977643069 C40.183006799643486,12.471727519928699 40.0752230567511,11.996192422164393 39.751871828073874,11.996192422164393 z" id="svg_3"/>
+	<path fill="#88C648" d="M25.793877123507535,12.260378587589008 c-0.05389187144620212,-0.15851169925476902 -0.16167561433860636,-0.2641861654246149 -0.3233512286772126,-0.2641861654246149 H0.30302192945394935 c-0.3233512286772126,0 -0.43113497156961683,0.47553509776430675 -0.1077837428924042,0.5812095639341532 L12.913719847865238,18.019636993845612 c0.3233512286772126,0.15851169925476902 0.5928105859082231,0.369860631594461 0.8083780716930314,0.6340467970190757 l17.353182605677077,23.988103820555033 c0.21556748578480833,0.2641861654246149 0.6467024573544252,0.10567446616984602 0.5928105859082231,-0.2641861654246149 L25.793877123507535,12.260378587589008 z" id="svg_4"/>
+	<path fill="#88C648" d="M14.422692248358898,8.508935038559475 l11.263401132256234,-3.540094616689841 c0.7544862002468291,-0.2641861654246149 0.7544862002468291,-0.47553509776430675 -0.05389187144620212,-0.5812095639341532 l-2.263458600740488,-0.211348932339692 l-2.640701700863903,-4.015629714454146 c-0.1077837428924042,-0.15851169925476902 -0.3233512286772126,-0.211348932339692 -0.5389187144620209,-0.10567446616984602 l-4.796376558711986,3.064559518925533 c-0.1077837428924042,0.10567446616984602 -0.21556748578480833,0.211348932339692 -0.16167561433860636,0.369860631594461 l0.6467024573544252,3.1173967520104564 l-1.6706480148322649,1.3209308271230746 C13.560422305219662,8.508935038559475 13.668206048112065,8.720283970899168 14.422692248358898,8.508935038559475 z" id="svg_5"/>
+</g></g></svg>
```

Comment:
The provided SVG contains a lot of data specific to the fortueno's website. Here's a version with the unnecessary data stripped out:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 39.63 42.51"><path fill="#88C648" d="M13.3 20.93c-.17-.22-.55-.16-.55.1L8.6 42.43c-.05.32.38.53.54.21l9.65-13.31c.16-.21.16-.53 0-.8l-5.5-7.6zM39.75 12H27.63c-.22 0-.33.15-.33.37l.92 4.75c.05.21.27.32.43.21l11.21-4.75c.32-.1.22-.58-.1-.58zM25.8 12.26c-.06-.16-.17-.26-.33-.26H.3c-.32 0-.43.47-.1.58l12.71 5.44c.33.16.6.37.81.63l17.36 24c.21.26.64.1.59-.27l-5.88-30.12zM14.42 8.5 25.7 4.98c.75-.27.75-.48-.06-.58l-2.26-.21L20.73.16c-.11-.16-.33-.21-.54-.1l-4.8 3.06c-.1.1-.21.21-.16.37l.65 3.12-1.67 1.32c-.65.58-.54.79.21.58z"/></svg>
```